content/consumer.ttl

```diff
@@ -2,7 +2,9 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :consumer
     skos:prefLabel "consumer"@en;
+    skos:related :integrator
     skos:definition '''
-    Someone who might read the output of an Orcfax datum but not necessarily integrate it, see integrator.
-    ''' ;
-    skos:related :integrator.
+      Someone who might read the output of an Orcfax datum but
```

Comment:
Sorry - I disagree with my own definition. 
It is literally "someone how 'consumes' an orcfax statement"

So maybe somehting like "someone how 'consumes' an orcfax statement. It is likely they are a participant in a tx in which orcfax data is used. It is therefore likely they have an interest in the orcfax protocol, in particular feed definition and data traceability. A consumer is not necessarily technical, and documentation should be aimed appropriately.:"

Reply:
> It is likely they are a participant in a tx in which orcfax data is used.

Not sure why it is likely? (Cerra used it as a data API first) 

> A consumer is not necessarily technical, and documentation should be aimed appropriately.:

Is this part of the definition you want or just some additional thinking/context? (if former, I'm not sure it should be))

Reply:
Moved debate to issue https://github.com/orcfax/glossary/issues/37